TESTS/mbedmicro-rtos-mbed/event_flags/main.cpp

```diff
@@ -45,20 +45,23 @@ using utest::v1::Case;
 #define PROHIBITED_FLAG 0x80000000   /* 10000000000000000000000000000000 */
 #define NO_FLAGS 0x0
 
-Semaphore sync_sem(0, 1);
-
 template<uint32_t flags, uint32_t wait_ms>
 void send_thread(EventFlags *ef)
 {
     for (uint32_t i = 0; i <= MAX_FLAG_POS; i++) {
         const uint32_t flag = flags & (1 << i);
         if (flag) {
             ef->set(flag);
-            ThisThread::sleep_for(wait_ms);
+            if (!is_ticker_used) {
```

Comment:
Can't this just be `if (wait_ms != 0)`? I think you're always in control of that via the template parameter, so I don't think you need a global here.